### PR TITLE
Add Tasks-related SPC bindings (start with `SPC :`)

### PR DESCRIPTION
**Pull request by o8vm**
Addressing the issue #236
All commands work fine.

### CODE REVIEW

**Review by stevenguh — COMMENTED** [review]
Thanks for submitting the PR. Just some minor discussion of some keys.

File: package.json [hunk 1]
```diff
@@ -316,6 +316,77 @@
...
+									},
+									{
+										"key": "t",
+										"name": "Run test task",
+										"icon": "beaker",
+										"type": "command",
+										"command": "workbench.action.tasks.test"
+									},
+									{
+										"key": "x",
```
[the remark this answers is not in the document]
Comment (o8vm) [reply package.json:375]: I would like to adopt `:`. Let's try it out. If we get feedback from other users, we'll consider changing it.

**Comment by o8vm** [comment]
Update:
`x` = Terminate Task
`:` = Run Task

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (stevenguh) [reply package.json:375]: Sounds good. Let's try that :)

**Pull request merged**